lib/path.cpp

```diff
@@ -249,3 +250,34 @@ bool Path::fileExists(const std::string &file)
     std::ifstream f(file.c_str());
     return f.is_open();
 }
+
+bool Path::isMatchingFilter(const std::string &filename, const std::string &filter)
+{
+    std::string searchString = filter;
+    size_t pos = filter.find('*');
+
+    // if the user added an asterix '*' replace it with regex '\w+' to match whole words
```

Comment:
if we have globbing with `*` then it should be consistent with how other globs are working. Imho it should be constistent with how it works on the command line.